### ISSUE TITLE
Clip4clip Performance on ViT B/16

### ISSUE THREAD

**Issue by JustinYuu** [princetonvisualai/MQVR#1]
Hi, 

Since the reported results of Clip4clip are based on ViT B/32, I wonder if you have tested the model's performance using ViT B/16? For my reproduced version, the performance on MSR-VTT achieves a significant boost (59.6, 72.7, 78.8 R@1 on RA, SA, and MF, respectively). However, the clip4clip model performance using ViT B/16 on MSVD is slightly lower than the reported results using ViT B/32, which is 48.2, 44.7, and 58.9 R@1 on RA, SA, and MF. Therefore, if you've tested the ViT B/16 performance, maybe I could directly use your reported results as baselines for comparison in case my reproducing process might be mistaken.

Thanks!

**Comment by zywangcode** [comment]
Hi, thanks for your interests in our work! We haven't experimented with ViT B/16 model so don't have the numbers available. Experimenting with ViT B/16 shouldn't require any change to the code base, just changing the `clip_model` field in the config file from "ViT-B/32" to "ViT-B/16" should do the work. Higher performance should be expected when using ViT B/16. If you are still getting lower performance, maybe you can try to tune the learning rate a bit as we only did limited hyperparameter tuning for the paper.
